src/alert-manager/src/cluster-utilization/send_alert.py

```diff
@@ -206,6 +206,9 @@ def send_alert(pai_url: str, cluster_usage, job_usage, user_usage):
             "cluster_usage": cluster_usage,
             "trigger_time": trigger_time,
         },
+        "annotations": {
+            "summary": "The weekly cluster usage has been reported, please check your email for details."
```

Comment:
I agree with the idea of adding a summary for the alerts generated here.

Two minor issues:
- remove `weekly` here, as the schedule is configurable
- we do not need to remind all the users to check their emails because not everyone receives the report

Reply:
1. DONE
2. In my observation, only the administrator account will receive all these types of alerts, and other users will not. The original intention of this PR is to address this comment: https://github.com/microsoft/pai/issues/5654#issuecomment-972806472.